### DISCUSSION TITLE
Is there a Qt Creator support planned?

### DISCUSSION THREAD

**Discussion by Desperado17** [ravenscroftj/turbopilot#66] [Q&A]
Greetings,

I'd like to use turbopilot to generate unit test code snippets for developers that use qt creator. I such a plugin for turbopilot planned or available?

Regards.

**Comment by ravenscroftj** [comment]
Hey - so whilst QTCreator isn't specifically something I'm familiar, it looks like there is already a [Qt Creator Github Copilot plugin](https://doc.qt.io/qtcreator/creator-copilot.html) that uses the neovim plugin for Copilot and I've seen some efforts to get that working with Fauxpilot [here](https://github.com/fauxpilot/fauxpilot/discussions/72?sort=top). I think that's probably the way forward for this use case.

I don't know when I'll have time to look into it myself but PRs always welcome!
